src/mantine-hooks/src/use-focus-trap/tabbable.ts

```diff
@@ -10,6 +10,15 @@ function hidden(element: HTMLElement) {
 }
 
 function visible(element: HTMLElement) {
```

Comment:
[This whole file could probably be re-written and only use CSS selectors](https://stackoverflow.com/a/30753870). But I am not sure about why all this was written the way it was, and the use cases.